flink-runtime/src/test/java/org/apache/flink/runtime/rest/messages/JobVertexDetailsInfoTest.java

```diff
@@ -83,10 +83,12 @@ protected JobVertexDetailsInfo getTestResponseInstance() throws Exception {
                         jobVertexMetrics,
                         "taskmanagerId3"));
 
+        int parallelism = random.nextInt();
         return new JobVertexDetailsInfo(
                 new JobVertexID(),
                 "jobVertex" + random.nextLong(),
-                random.nextInt(),
+                parallelism,
+                2 * parallelism,
```

Comment:
This could overflow leading to a negative maxParallelism which may break an assertion somewhere at some point

Reply:
Good point!

Reply:
Done